Onboarding/TechicalOverview.md

```diff
@@ -0,0 +1,30 @@
+# CanDIG Technical Onboarding
+## CanDIG Project Overview
+The CanDIG project has a distributed architecture, meaning each site is meant to run their own _____(What do you call everything represented in the diagram together) and there is no centralized CanDIG server. Users can use their local CanDIG _____(^) to communicate with other sites. Every site is responsible for handling the authentication of it's users and the authorization of incoming requests from other sites. CanDIG uses an API driven approach, where users send requests to their CanDIG which communicate with other sites through ...?
```

Comment:
> (What do you call everything represented in the diagram together)

**Great question!** I have in the past called it `CanDIG Suite`. I have heard `Site` be used. As noted, term `project` also has been used but these terms are overloaded as everything in computer science. [insert cache-invalidation joke here].
- I say we use `CanDIG Platform` when talking about our CanDIG project because there are "projects" that will use CanDIG as a platform.
- Each of those projects will have multiple "sites". So that settles that part for me. Maybe the local CanDIG Suite can be called `Local Deployment? But that is another level of overloading.
- This still does not answer your question and hence we need input from @ljdursi et. al. on this.

> authentication of it's users
- Minor correction: its

> CanDIG uses an API driven approach, where users send requests to their CanDIG which communicate with other sites through ...?
- their CanDIG "deployment"? which 
- through Federation API

Others: Please feel free to correct anything above and chime in please.

Reply:
"suite", or "stack" is good.

A "site" is a running instance of the suite/stack, presumably with some data under their control, and might include the people operating it.  "Deployment" might be a synonym?

The "platform" is the network of federated sites and the agreements/protocols that knit them together.

"Project" is our somewhat unsatisfactory term for a health data/sequencing effort whose data is hosted at the sites and using the platform to do their work.

Those terms (and indeed the concepts) are still evolving - don't take the above as cast in stone!